### PR TITLE
fix `amount` deserialization  + examples minor fixes

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

- fix: `amount` deserialization which should be a string instead of u64
- examples minor fixes

## Links to any relevant issues

part of #388 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/examples/how_tos/native_tokens/send.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const RECV_ADDRESS =
 // In this example we will send native tokens.
 //
 // Make sure that `STRONGHOLD_SNAPSHOT_PATH` and `WALLET_DB_PATH` already exist by
-// running the `how_tos/accounts-and-addresses/create-wallet` example!
+// running the `how_tos/accounts-and-addresses/create-account` example!
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/how_tos/native_tokens/send.ts]: `accounts-and-addresses` is supposed to have underscores, not dashes

Comment (qrayven) [reply bindings/nodejs/examples/how_tos/native_tokens/send.ts]: good point, fixed

File: bindings/nodejs/examples/client/07-get-block-data.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ require('dotenv').config({ path: '.env' });
 // Run with command:
 // yarn run-example ./client/07-get-block-data.ts
 
-// In this example we will send a block and get the data and metadata for it
+// In this example we will send a block and get the data and metadata for the block.
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/client/07-get-block-data.ts]: I think it was fine tbh, bit repetitive now 😅

File: bindings/nodejs/examples/exchange/0-generate-mnemonic.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@
 
 import { Utils } from '@iota/sdk';
 
+// In this example we will generate a new random mnemonic.
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/exchange/0-generate-mnemonic.ts]: There is already a description, remove one of them?

File: bindings/nodejs/examples/exchange/1-create-account.ts [hunk 4]
```diff
@@ -10,6 +10,7 @@ import { Wallet, WalletOptions, CoinType } from '@iota/sdk';
 // This example uses secrets in environment variables for simplicity which should not be done in production.
 require('dotenv').config({ path: '.env' });
 
+// In this example we will create a wallet from mnemonic and a new account.
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/exchange/1-create-account.ts]: There is already a description, remove one of them?

Comment (thibault-martinez) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: Same for all exchange examples

Comment (qrayven) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: omg,  I need some days off :sweat_smile: , fixed

File: bindings/nodejs/examples/client/07-get-block-data.ts (hunk 2 above)
Comment (qrayven) [reply bindings/nodejs/examples/client/07-get-block-data.ts]: fixed

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**